internal/service/environments/main.go

```diff
@@ -0,0 +1,17 @@
+package environments
+
+import (
+	"github.com/ucl-arc-tre/portal/internal/graceful"
+	"github.com/ucl-arc-tre/portal/internal/types"
+)
+
+func GetAll() ([]types.Environment, error) {
+	db := graceful.NewDB()
```

Comment:
this does the init call on every function call. I think we should use the standard pattern here of: the service is a struct https://github.com/ucl-arc-tre/portal/blob/068fe2c3338bff7ee087acb79046bf1ce7d0790f/internal/service/agreements/main.go#L16-L18
then the handler gets a new service https://github.com/ucl-arc-tre/portal/blob/068fe2c3338bff7ee087acb79046bf1ce7d0790f/internal/handler/web/main.go#L16-L17